isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -843,6 +844,21 @@ namespace Isis {
     return history;
   }
 
+  /**
+   * This method will read an OriginalLabel from a cube.
+   */
+  OriginalLabel Cube::readOriginalLabel() const {
+    Blob origLabelBlob("IsisCube", "OriginalLabel");
+    try {
+      origLabelBlob.Read(fileName());
+    }
+    catch(IException &) {
+     //Intentionally left blank
+    }
```

Comment:
Unlike History, we want to actually throw an error here if we can't read the original label from the file.

Reply:
Is there a specific error that should be thrown, or should I just remove the try/catch so that the original error is raised?

Reply:
I would append something like "Cube not read the original label from [cube.cub]." and then rethrow.